tensorboard/defs/web.bzl

```diff
@@ -377,3 +379,60 @@ tf_web_library = rule(
         "_closure_library_base": CLOSURE_LIBRARY_BASE_ATTR,
     }.items()),
     outputs=DEPRECATED_CLUTZ_OUTPUTS)
+
+def _tf_web_test_python_shell(ctx):
+  ctx.actions.expand_template(
+      template=ctx.file._template,
+      output=ctx.outputs.main,
+      substitutions={
+          "{BINARY_PATH}": ctx.executable.web_library.short_path,
+          "{WEB_PATH}": ctx.attr.web_path,
+      }
+  )
+
+tf_web_test_python_shell = rule(
```

Comment:
Unless we see this being used outside this file, prefer to prefix with underscore (and I guess move the impl def to a _impl suffix or something).

Also, I might replace "shell" with "stub" or "wrapper" since a "python shell" sounded like an actual python interpreter shell on first read to me.